docs/migration-guides/1.11-1.12.md

```diff
@@ -13,4 +13,164 @@ During migration, it may turn out that some resource specifications are too comp
 Here are the steps you need to follow to ensure your services are properly migrated:
 
 1. [Verify the migration](https://github.com/kyma-project/kyma/blob/1.12/docs/api-gateway/03-04-migration.md#verify-the-automatic-migration). 
-2. If you can still see any Api CRs in use, use the [manual migration](https://github.com/kyma-project/kyma/blob/1.12/docs/api-gateway/03-04-migration.md#manual-migration) guide to migrate them.
\ No newline at end of file
+2. If you can still see any Api CRs in use, use the [manual migration](https://github.com/kyma-project/kyma/blob/1.12/docs/api-gateway/03-04-migration.md#manual-migration) guide to migrate them.
+
+## Serverless
+
+Lambdas (Functions) created in previous releases won't work anymore because their underlying component, Kubeless, was removed. Migration from the previous solution to the new one requires manual steps - because of the complexity of Kubeless configuration, it was not possible to migrate the functions automatically.
+
+Follow these sections and steps to migrate your existing Kubeless-based Functions to the new KService-based ones.
+
+>**TIP:** All examples in this section provide references to the corresponding fields from the deprecated resources. For example, `source: {spec.function}` means that you must enter the value from the **spec.function** field of the deprecated Function in the **spec.source** field of the new Function:
+> ```bash
+> spec:
+>   ...
+>   source: {spec.function}
+> ```
+
+### Migrate functions.kubeless.io to functions.serverless.kyma-project.io
+
+1. Remove `kubeless.io/function` from the **metadata.finalizers** field. To do that, edit the deprecated Function:
+
+    ```bash
+    kubectl edit functions.kubeless.io {NAME} -n {NAMESPACE}
```

Comment:
**nit:** `kubectl patch functions.kubeless.io {NAME} --type=merge -p '{"metadata":{"finalizers":null}}'` 

with that approach user only needs to specify the resource name, nothing more so IMO it's better for copy-paste. What's more, you will ensure that the finalizer list is really empty as this is your goal.